Queueomatic/Server/Endpoints/ForgetPassword/ForgetPasswordEndpoint.cs

```diff
@@ -0,0 +1,58 @@
+﻿using FastEndpoints;
+using Queueomatic.DataAccess.UnitOfWork;
+using Queueomatic.Server.Services.AuthenticationService;
+using Queueomatic.Server.Services.MailService;
+using Queueomatic.Shared.DTOs;
+
+
+namespace Queueomatic.Server.Endpoints.ForgetPassword;
+
+public class ForgetPasswordEndpoint : Endpoint<ForgetPasswordRequest>
+{
+    private readonly IAuthenticationService _authenticationService;
+    private readonly IConfiguration _configuration;
+    private readonly IUnitOfWork _unitOfWork;
+    private readonly IMailService _mailService;
+    public ForgetPasswordEndpoint(IAuthenticationService authenticationService, IConfiguration configuration, IUnitOfWork unitOfWork, IMailService mailService)
+    {
+        _authenticationService = authenticationService;
+        _configuration = configuration;
+        _unitOfWork = unitOfWork;
+        _mailService = mailService;
+    }
+
+    public override void Configure()
+    {
+        Post("/forgetPassword");
```

Comment:
The path should be something like `/forgotPassword` or `/sendPasswordResettingEmail`

Reply:
The first one in that case.